install_config/http_proxies.adoc

```diff
@@ -44,8 +44,8 @@ endif::[]
 The `NO_PROXY` environment variable lists all of the {product-title}
 components and all IP addresses that are managed by {product-title}.
 
-`NO_PROXY` accepts a comma-separated list of hosts, IP addresses, or IP ranges
-in CIDR format:
+In all cases except for Docker, `NO_PROXY` accepts a comma-separated list of
```

Comment:
Lets not say in "all cases" as CIDR format is special only to the OpenShift Master and Node Services.  We added CIDR functionality here in the PR https://github.com/kubernetes/kubernetes/pull/23003

I am not entirely sure but CIDR format is not supported when setting no_proxy variable in the builds and git.  It can be set for the pod environment variable but the support for CIDR would be dependant on the service using that variable, which in most cases its not supported.